Sources/VitalCore/Core/Client/Data Models/Patches/QuantitySample.swift

```diff
@@ -55,10 +59,33 @@ public struct LocalQuantitySample: Hashable, Encodable {
       sourceBundle: sourceBundle,
       productType: productType,
       type: type,
-      unit: unit
+      unit: unit,
+      metadata: metadata
     )
   }
 
+  public static func == (lhs: LocalQuantitySample, rhs: LocalQuantitySample) -> Bool {
+    lhs.value == rhs.value &&
+    lhs.startDate == rhs.startDate &&
+    lhs.endDate == rhs.endDate &&
+    lhs.sourceBundle == rhs.sourceBundle &&
+    lhs.productType == rhs.productType &&
+    lhs.type == rhs.type &&
+    lhs.unit == rhs.unit &&
+    lhs.metadata == lhs.metadata
+  }
+
+  public func hash(into hasher: inout Hasher) {
+    hasher.combine(value)
+    hasher.combine(startDate)
+    hasher.combine(endDate)
+    hasher.combine(sourceBundle)
+    hasher.combine(productType)
+    hasher.combine(type)
+    hasher.combine(unit)
+    hasher.combine(metadata)
+  }
```

Comment:
You can remove these. All properties are Hashable, so the compiler should be able to synthesise these.